### PR TITLE
Add field for textbook content

### PR DESCRIPTION
**Pull request by brinxmat**
Added field for textbook to ChapterArticle and PeerReviewedMonograph

Sarah: se gjerne på documentation/{Types where we expect the textbookContent property}

### CODE REVIEW

File: documentation/BookAnthology.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "type" : "Publication",
-  "identifier" : "5a0d2a7d-d0f3-4d3c-918f-067d6a37eef7",
+  "identifier" : "955fab17-542d-44fa-a6d0-a2100d8b9ef3",
```
Comment (janbj) [inline documentation/BookAnthology.json:3]: Do we need id as part of the ontology for Publication or is that implicit?

**Review by janbj — APPROVED** [review]

File: documentation/DegreePhd.json [hunk 2]
```diff
@@ -10,16 +10,17 @@
...
+  "publishedDate" : "2020-09-23T09:40:34.473624Z",
+  "indexedDate" : "2020-09-23T09:40:34.473605Z",
   "handle" : "https://example.org/fakeHandle/13213",
   "doi" : "https://example.org/yet/another/fake/doi/1231/12311",
   "doiRequest" : {
     "type" : "DoiRequest",
     "status" : "APPROVED",
-    "date" : "2020-09-18T07:35:55.134450Z"
+    "date" : "2020-09-23T09:40:34.473494Z",
+    "messages" : [ ]
```
Comment (sarahse) [inline documentation/DegreePhd.json:23]: hva er dette feltet for?

Comment (sarahse) [reply documentation/DegreePhd.json:23]: nvm så ikke at det var inni doiRequests-objektet

**Review by sarahse — APPROVED** [review]

File: documentation/BookAnthology.json (hunk 1 above)
Comment (brinxmat) [reply documentation/BookAnthology.json:3]: In essence, the nature of the identifier is not important, however, we have platform expectations, so it becomes important. I will create a code example as a suggested fix for this.

**Review by norrs — APPROVED** [review]

**Comment by norrs** [comment]
Only confusing with this pull review is maybe actually finding what is the new changes (as we get diffs with all timestamps and random generated identifiers).  The diff is really a lot smaller :p

**Pull request merged**